web_last_viewed_records/static/src/js/main.js

```diff
@@ -0,0 +1,178 @@
+openerp.web_last_viewed_records = function(instance){
```

Comment:
I think it should be discussed. Odoo's point_of_sale doesn't test existence of localStorage. https://github.com/odoo/odoo/blob/master/addons/point_of_sale/static/src/js/db.js
Is it a bug in odoo or there are some logic in it?